dali/sasha/saxref.cpp

```diff
@@ -344,8 +344,7 @@ struct cDirDesc
         unsigned pf;
         StringAttr mask;
         const char *fn = decodeName(drv,name,node,numnodes,mask,pf,nf);
-        bool misplaced = !grp.queryNode(pf).endpoint().equals(ep);
-
+        bool misplaced = nf!=grp.ordinality() || pf>=grp.ordinality() || !grp.queryNode(pf).endpoint().equals(ep);
```

Comment:
I'm not familiar with this code - which group it applies to.
One concern is what if the ordinality doesn't equal the group size e.g.,
create file, copy to a diff size cluster, delete from the original cluster.

Reply:
The file representations it's building up here are candidates.
There is a later stage where if it finds there're complete and in Dali, they will be ignored.